Classes/Settings.cs

```diff
@@ -490,5 +481,32 @@ protected static void SetValue(string propertyName, string value)
                 }
             }
         }
+
+        private static string GetValuePostfix(Model.Programme progInfo, string name, string defaultValue)
```

Comment:
Having overloads of functions that are disparate concerning internal and private feels like stretching the concept too far. It's technically not wrong though. The compiler doesn't mind.